admin/class-yoast-form.php

```diff
@@ -583,24 +590,27 @@ public function toggle_switch( $var, $values, $label ) {
 			$this->options[ $var ] = 'off';
 		}
 
+		$help_class = ! empty( $help ) ? ' switch-container__has-help' : '';
+
 		$var_esc = esc_attr( $var );
 
-		echo '<div class="switch-container">';
-		echo '<fieldset id="', $var_esc, '" class="fieldset-switch-toggle"><legend>', $label, '</legend>
-		<div class="switch-toggle switch-candy switch-yoast-seo">';
+		echo "<div class='switch-container$help_class'>";
+		echo '<fieldset id="', $var_esc, '" class="fieldset-switch-toggle"><legend>', $label, '</legend>', $help,
+		'<div class="switch-toggle switch-candy switch-yoast-seo">';
 
 		foreach ( $values as $key => $value ) {
-			$screen_reader_text = '';
+			$screen_reader_text = $screen_reader_text_html = '';
```

Comment:
Please place each definition on their own line.

Reply:
OK (not sure why :) )